docs/it/index.md

```diff
@@ -140,19 +140,19 @@ learning universitario.
     </tr>
 <!-- =============================== WEEK 6 ================================ -->
     <tr>
-      <td rowspan="3" align="center"><a href="en/week06/06">⑥</a></td>
-      <td rowspan="2">Lecture</td>
-      <td><a href="en/week06/06-1">Applicazioni di reti convoluzionali</a></td>
+      <td rowspan="3" align="center"><a href="it/week06/06">⑥</a></td>
+      <td rowspan="2">Lezione</td>
+      <td><a href="{{site.baseurl}}/it/week06/06-1">Applicazioni di reti convoluzionali</a></td>
       <td rowspan="2">
         <a href="https://drive.google.com/open?id=1opT7lV0IRYJegtZjuHsKhlsM5L7GpGL1">🖥️</a>
         <a href="https://drive.google.com/open?id=1sdeVBC3nuh5Zkm2sqzdScEicRvLc_v-F">🖥️</a>
         <a href="https://youtu.be/ycbMGyCPzvE">🎥</a>
       </td>
     </tr>
-    <tr><td><a href="en/week06/06-2">RNNs, GRUs, LSTMs, Attenzione, Seq2Seq, e Reti di Memoria</a></td></tr>
+    <tr><td><a href="it/week06/06-2">RNNs, GRUs, LSTMs, Attenzione, Seq2Seq, e Reti di Memoria</a></td></tr>
     <tr>
-      <td rowspan="1">Practicum</td>
-      <td><a href="en/week06/06-3">Architettura delle RNN e modelli LSTM</a></td>
+      <td rowspan="1">Practica</td>
```

Comment:
Pratica? Non c'è la "c" prima della "t" in italiano 😝

Reply:
😅 too much translation, now I'm speaking Itanglish